.github/workflows/test-suite.yml

```diff
@@ -143,3 +143,71 @@ jobs:
           status: ${{ job.status }}
           fields: job,commit,ref,eventName,author,took
           author_name: Test Suite (iOS)
+
+  android:
+    runs-on: macos-latest
```

Comment:
Do we need a macOS for an Android test?

Reply:
That is a good question. Android emulators on Linux seems to be slow on GitHub runners as no hardware acceleration support.
See https://github.com/ReactiveCircus/android-emulator-runner#can-i-use-this-action-on-linux-vms
But it maybe worth to try. Please let me try later tomorrow for CI workers are not busy then.

Reply:
Some update for ubuntu runner. I've tried a l lot on the past weekend.
The emulator always have the `System UI isn't responding` issue and detox test cannot be installed.
Moreover, it take 15 minutes to boot up the emulator.
I would say android emulator on github ubuntu runner is still not feasible right now.